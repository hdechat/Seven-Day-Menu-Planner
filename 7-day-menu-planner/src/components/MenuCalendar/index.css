.week {
  display: flex;
  height: 635px;
  justify-content: space-between;
  list-style-type: none;
  margin: 2rem auto;
  padding: 0rem;
  width: 98%;
}

.menu h2 {
  margin: 0;
}

li {
  margin: 0 1rem;
  width: 8.7rem;
  height: 30rem;
}

h3 {
  color: ghostwhite;
  font-weight: lighter;
  letter-spacing: 0.4rem;
  margin: 0;
}

@media screen and (max-width: 350px) {
  h3 {
    font-size: 30px;
    margin-top: 5rem;
  }

  h6 {
    font-size: 26px;
  }

  li {
    height: fit-content;
  }

  .week {
    display: block;
  }

  .day {
    width: 385%;
  }
}