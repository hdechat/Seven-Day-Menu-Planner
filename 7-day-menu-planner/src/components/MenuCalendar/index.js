import React from 'react';
import Day from '../Day';

const MenuCalendar = ({menu}) => {
  console.log(menu)
  return(
    <article>
      <h1>THIS WEEKS MENU PLAN</h1>
      <ul>
        <li>Sunday<Day menu={menu.sunday}/></li>
        <li>Monday<Day menu={menu.monday}/></li>
        <li>Tuesday<Day menu={menu.tuesday}/></li>
        <li>Wednesday<Day menu={menu.wednesday}/></li>
        <li>Thursday<Day menu={menu.thursday}/></li>
        <li>Friday<Day menu={menu.friday}/></li>
        <li>Saturday<Day menu={menu.saturday}/></li>
      </ul>
    </article>
  )
}

export default MenuCalendar;
