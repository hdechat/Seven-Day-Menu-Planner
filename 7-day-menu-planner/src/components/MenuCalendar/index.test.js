import React from 'react';
import MenuCalendar from './index.js';
import { shallow } from 'enzyme';

describe('Add To Menu', () => {
  let wrapper;
  let mockMenu;

  beforeEach(() => {
    mockMenu = {
      menu: {
        breakfast: {title: 'Recipe Name'}
      }
    }

    wrapper = shallow(<MenuCalendar menu={mockMenu} />);
  });

  it('matches snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});