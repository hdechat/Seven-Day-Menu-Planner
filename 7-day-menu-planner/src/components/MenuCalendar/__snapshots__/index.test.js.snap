// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add To Menu matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MenuCalendar
    menu={
        Object {
            "menu": Object {
              "breakfast": Object {
                "title": "Recipe Name",
              },
            },
          }
    }
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <ul
        className="week"
>
        <li>
                <h3>
                        Sunday
                </h3>
                <withRouter(Connect(Day))
                        day="sunday"
                />
        </li>
        <li>
                <h3>
                        Monday
                </h3>
                <withRouter(Connect(Day))
                        day="monday"
                />
        </li>
        <li>
                <h3>
                        Tuesday
                </h3>
                <withRouter(Connect(Day))
                        day="tuesday"
                />
        </li>
        <li>
                <h3>
                        Wednesday
                </h3>
                <withRouter(Connect(Day))
                        day="wednesday"
                />
        </li>
        <li>
                <h3>
                        Thursday
                </h3>
                <withRouter(Connect(Day))
                        day="thursday"
                />
        </li>
        <li>
                <h3>
                        Friday
                </h3>
                <withRouter(Connect(Day))
                        day="friday"
                />
        </li>
        <li>
                <h3>
                        Saturday
                </h3>
                <withRouter(Connect(Day))
                        day="saturday"
                />
        </li>
</ul>,
      "className": "menu",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <li>
            <h3>
                        Sunday
            </h3>
            <withRouter(Connect(Day))
                        day="sunday"
            />
</li>,
          <li>
            <h3>
                        Monday
            </h3>
            <withRouter(Connect(Day))
                        day="monday"
            />
</li>,
          <li>
            <h3>
                        Tuesday
            </h3>
            <withRouter(Connect(Day))
                        day="tuesday"
            />
</li>,
          <li>
            <h3>
                        Wednesday
            </h3>
            <withRouter(Connect(Day))
                        day="wednesday"
            />
</li>,
          <li>
            <h3>
                        Thursday
            </h3>
            <withRouter(Connect(Day))
                        day="thursday"
            />
</li>,
          <li>
            <h3>
                        Friday
            </h3>
            <withRouter(Connect(Day))
                        day="friday"
            />
</li>,
          <li>
            <h3>
                        Saturday
            </h3>
            <withRouter(Connect(Day))
                        day="saturday"
            />
</li>,
        ],
        "className": "week",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3>
                Sunday
</h3>,
              <withRouter(Connect(Day))
                day="sunday"
/>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Sunday",
              },
              "ref": null,
              "rendered": "Sunday",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "day": "sunday",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3>
                Monday
</h3>,
              <withRouter(Connect(Day))
                day="monday"
/>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Monday",
              },
              "ref": null,
              "rendered": "Monday",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "day": "monday",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3>
                Tuesday
</h3>,
              <withRouter(Connect(Day))
                day="tuesday"
/>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Tuesday",
              },
              "ref": null,
              "rendered": "Tuesday",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "day": "tuesday",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3>
                Wednesday
</h3>,
              <withRouter(Connect(Day))
                day="wednesday"
/>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Wednesday",
              },
              "ref": null,
              "rendered": "Wednesday",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "day": "wednesday",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3>
                Thursday
</h3>,
              <withRouter(Connect(Day))
                day="thursday"
/>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Thursday",
              },
              "ref": null,
              "rendered": "Thursday",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "day": "thursday",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3>
                Friday
</h3>,
              <withRouter(Connect(Day))
                day="friday"
/>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Friday",
              },
              "ref": null,
              "rendered": "Friday",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "day": "friday",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "li",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <h3>
                Saturday
</h3>,
              <withRouter(Connect(Day))
                day="saturday"
/>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Saturday",
              },
              "ref": null,
              "rendered": "Saturday",
              "type": "h3",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "day": "saturday",
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "li",
        },
      ],
      "type": "ul",
    },
    "type": "section",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <ul
          className="week"
>
          <li>
                    <h3>
                              Sunday
                    </h3>
                    <withRouter(Connect(Day))
                              day="sunday"
                    />
          </li>
          <li>
                    <h3>
                              Monday
                    </h3>
                    <withRouter(Connect(Day))
                              day="monday"
                    />
          </li>
          <li>
                    <h3>
                              Tuesday
                    </h3>
                    <withRouter(Connect(Day))
                              day="tuesday"
                    />
          </li>
          <li>
                    <h3>
                              Wednesday
                    </h3>
                    <withRouter(Connect(Day))
                              day="wednesday"
                    />
          </li>
          <li>
                    <h3>
                              Thursday
                    </h3>
                    <withRouter(Connect(Day))
                              day="thursday"
                    />
          </li>
          <li>
                    <h3>
                              Friday
                    </h3>
                    <withRouter(Connect(Day))
                              day="friday"
                    />
          </li>
          <li>
                    <h3>
                              Saturday
                    </h3>
                    <withRouter(Connect(Day))
                              day="saturday"
                    />
          </li>
</ul>,
        "className": "menu",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <li>
              <h3>
                            Sunday
              </h3>
              <withRouter(Connect(Day))
                            day="sunday"
              />
</li>,
            <li>
              <h3>
                            Monday
              </h3>
              <withRouter(Connect(Day))
                            day="monday"
              />
</li>,
            <li>
              <h3>
                            Tuesday
              </h3>
              <withRouter(Connect(Day))
                            day="tuesday"
              />
</li>,
            <li>
              <h3>
                            Wednesday
              </h3>
              <withRouter(Connect(Day))
                            day="wednesday"
              />
</li>,
            <li>
              <h3>
                            Thursday
              </h3>
              <withRouter(Connect(Day))
                            day="thursday"
              />
</li>,
            <li>
              <h3>
                            Friday
              </h3>
              <withRouter(Connect(Day))
                            day="friday"
              />
</li>,
            <li>
              <h3>
                            Saturday
              </h3>
              <withRouter(Connect(Day))
                            day="saturday"
              />
</li>,
          ],
          "className": "week",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3>
                  Sunday
</h3>,
                <withRouter(Connect(Day))
                  day="sunday"
/>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Sunday",
                },
                "ref": null,
                "rendered": "Sunday",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "day": "sunday",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3>
                  Monday
</h3>,
                <withRouter(Connect(Day))
                  day="monday"
/>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Monday",
                },
                "ref": null,
                "rendered": "Monday",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "day": "monday",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3>
                  Tuesday
</h3>,
                <withRouter(Connect(Day))
                  day="tuesday"
/>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Tuesday",
                },
                "ref": null,
                "rendered": "Tuesday",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "day": "tuesday",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3>
                  Wednesday
</h3>,
                <withRouter(Connect(Day))
                  day="wednesday"
/>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Wednesday",
                },
                "ref": null,
                "rendered": "Wednesday",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "day": "wednesday",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3>
                  Thursday
</h3>,
                <withRouter(Connect(Day))
                  day="thursday"
/>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Thursday",
                },
                "ref": null,
                "rendered": "Thursday",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "day": "thursday",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3>
                  Friday
</h3>,
                <withRouter(Connect(Day))
                  day="friday"
/>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Friday",
                },
                "ref": null,
                "rendered": "Friday",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "day": "friday",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "li",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h3>
                  Saturday
</h3>,
                <withRouter(Connect(Day))
                  day="saturday"
/>,
              ],
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Saturday",
                },
                "ref": null,
                "rendered": "Saturday",
                "type": "h3",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "day": "saturday",
                },
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
            ],
            "type": "li",
          },
        ],
        "type": "ul",
      },
      "type": "section",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
